_posts/2015-02-19-1-managing-datasources.md

```diff
@@ -89,6 +89,21 @@ Notes:
 1. To obtain BigQuery credentials follow the guidelines at: https://developers.google.com/bigquery/authorization#service-accounts
 2. You need to install the `google-api-python-client`, `oauth2client` and `pyopenssl` packages (PyOpenSSL requires `libffi-dev` and `libssl-dev` packages), as they are not included in the `requirements.txt` file.
 
+### Google Spreadsheets
+
+* **Type**: google_spreadsheets
+* **Options**:
+	* Credentials filename (credfilePath) (mandatory)
+* **Options string format (for v0.5 and older)**: {"credfilePath" : "/somewhere/23fjkfjdsfj21312-privatekey.p12"}
+
+Notes:
+
+1. To obtain Google ServiceAccount credentials follow the guidelines at: https://developers.google.com/console/help/new/#serviceaccounts

```

Comment:
The final part mentions both p12 file and JSON... 

Also don't forget to update here the name of the param.